recipes/_start.rb

```diff
@@ -13,7 +13,8 @@
 package_version = Gem::Version.new(node['splunk']['package']['version'])
 command = "#{node['splunk']['cmd']} enable boot-start -user #{node['splunk']['user']}"
 command += " -group #{node['splunk']['group']}" if package_version >= Gem::Version.new('7.3.0')
-command += " #{node['splunk']['boot_start_args']}" if package_version >= Gem::Version.new('7.2.2')
+command += " #{node['splunk']['boot_start_args']} -systemd-unit-file-name #{node['splunk']['systemd_unit_file_name']}" if node['platform_version'].to_i != 6 && (package_version >= Gem::Version.new('7.2.2'))
+command += " #{node['splunk']['boot_start_args']}" if node['platform_version'].to_i == 6 && (package_version >= Gem::Version.new('7.2.2'))
```

Comment:
```suggestion
command += " #{node['splunk']['boot_start_args']}" if package_version >= Gem::Version.new('7.2.2')
command += "  -systemd-unit-file-name #{node['splunk']['systemd_unit_file_name']}" if node['platform_version'].to_i > 6 && (package_version >= Gem::Version.new('7.2.2'))
```

Do we also need to worry about the platform family here (i.e. windows)?  Previously, we only set systemd-managed 1 and the systemd-unit-file in the default boot start args when the platform family was rhel,fedora,or amazon and the platform_version was 7 or 8.  This only checks the version.

Reply:
When the platform version is 6 it doesn't want to use systemd according to rspec.

Reply:
nevermind about the windows bit, the guard on line 20 takes care of that.  But I still think the code suggestion I provided is a bit cleaner since it doesn't repeat the boot_start_args.

Reply:
Cleaned: https://github.com/cerner/cerner_splunk/pull/265/commits/96382c0de99b55664daaf8e4da926ea4029fda4e

2 splunk installs tests and rspec still passes as well.